docs/modules/plugins/partials/locator-header.adoc

```diff
@@ -2,7 +2,7 @@
 
 === Description
 
-Describes elements locator. Should match regular expression: `(?:By\\.)?([a-zA-Z]+)\\((.+?)\\):?([a-zA-Z]*)?`
+Describes elements locator.
 
 === Syntax
```

Comment:
since the section above is going to be removed, the title for this section can be removed as well